server/www/templates/posts/modify/post-editor.html

```diff
@@ -31,11 +31,12 @@ <h1 class="mode-context-title">{{post.form.name}}</h1>
      <div class="main-col">
 
           <div class="form-sheet">
+              <div class="post-band" style="background-color: {{form.color}}"></div>
               <div class="form-field title form-field-adapt init"
               adaptive-form
               ng-class="{
-                'error': form.title.$invalid && form.title.$dirty,
-                'success': !form.title.$invalid && form.title.$dirty
+                'error': postForm.title.$invalid && postForm.title.$dirty,
```

Comment:
This error happens many times further down. Can you fix there too? (form -> postForm)